src/random.cr

```diff
@@ -289,6 +289,41 @@ module Random
     end
   end
 
+  {% for type, values in {
+                           "Int8".id   => %w(20 -66 89 19),
+                           "UInt8".id  => %w(186 221 127 245),
+                           "Int16".id  => %w(-32554 32169 -20152 -7686),
+                           "UInt16".id => %w(39546 44091 2874 17348),
+                           "Int32".id  => %w(1870830079 -1043532158 -867180637 -1216773590),
+                           "UInt32".id => %w(3147957137 4245108745 2207809043 3184391838),
+                           "Int64".id  => %w(4438449217673515190 8514493061600538358 -4874671083204037318 -7825896160729246667),
+                           "UInt64".id => %w(15004487597684511003 12027825265648206103 11303949506191212698 6228566501671148658),
+                         } %}
+    # Returns a random {{type}}
+    #
+    # ```
+    # rand({{type}}) # => {{values[0].id}}
+    # ```
+    def rand(type : {{type}}.class) : {{type}}
+      rand_type_from_bytes(type)
```

Comment:
It's strange to have a separate implementation for this when `rand(type::MIN..type::MAX)` works fine.

Also kind of funny that `random_bytes` unrolls integers into bytes (and inverts for endinanness) and then this rolls them back into an integer (without inverting for endianness - though double-inverting would also be weird).

So yes, this implementation is endianness-dependent, unlike everything else.

And then, just by chance,
`Random.new(x).rand(Int32::MIN..Int32::MAX) == Random.new(x).rand(Int32)`
but
`Random.new(x).rand(Int64::MIN..Int64::MAX) != Random.new(x).rand(Int64)`